libs/icons/src/index.ts

```diff
@@ -91,4 +93,5 @@ export * from './Alert';
 export * from './PartyFill';
 
 // Tangle Icons
+export * from './Tangle';
 export * from './TangleIcon';
```

Comment:
Could you please clarify why we have two `Tangle` icons present here? Wouldn't it be more efficient to use `<ChainIcon name="tangle" />` instead?